src/languageservice/services/validation/unused-anchors.ts

```diff
@@ -39,13 +44,29 @@ export class UnusedAnchorsValidator implements AdditionalValidator {
             document.positionAt(aToken.offset),
             document.positionAt(aToken.offset + aToken.source.length)
           );
-          const warningDiagnostic = Diagnostic.create(range, `Unused anchor "${aToken.source}"`, DiagnosticSeverity.Hint, 0);
+          const warningDiagnostic = Diagnostic.create(
+            range,
+            `Unused anchor "${aToken.source}"`,
+            DiagnosticSeverity.Information,
+            0
+          );
           warningDiagnostic.tags = [DiagnosticTag.Unnecessary];
           result.push(warningDiagnostic);
         }
       }
     }
 
+    unIdentifiedAlias.forEach((node) => {
+      const nodeRange = node.range;
+      if (nodeRange) {
+        const startOffset = nodeRange[0];
+        const endOffset = nodeRange[1];
+        const range = Range.create(document.positionAt(startOffset), document.positionAt(endOffset));
+        const warningDiagnostic = Diagnostic.create(range, `Unresolved alias "${node}"`, DiagnosticSeverity.Information, 0);
```

Comment:
Do you think it's worth adding "it will be treated as a string instead" into the error message?

Reply:
No basically yaml treated as string. I feel no more additional messages